.storybook/preview-head.html

```diff
@@ -0,0 +1,12 @@
+<style>
+    body {
+        font-family: 'Fira Mono', monospace;
+        color: '#fff';
+    }
+    .sbdocs-p code,
+    .sbdocs-li code {
+        color: #fff !important;
```

Comment:
I don't think we use pure white anywhere on the site. The only exception I see is in `link.js` where I think I had a placeholder color that never got updated. I see a bunch of `#fff` in the SB theme, but that doesn't reflect the color we use.